### ISSUE TITLE
Wifi no working in Ubuntu 18.04LTS 

### ISSUE THREAD

**Issue by mad03day** [cvetaevvitaliy/stm32mp1-ubuntu#4]
![image](https://user-images.githubusercontent.com/71565506/168282395-1fd3a87c-14ee-4186-94b7-ac0872307b55.png)

**Comment by cvetaevvitaliy** [comment]
Soon, I will prepare an update build-script for building Kernel  from the official ST repository `https://github.com/STMicroelectronics/linux` 

Thank you for testing and interest in my project

**Comment by cvetaevvitaliy** [comment]
Hi! I fixed WiFi and BLE

```
[   16.562404] Bluetooth: hci0: BCM: chip id 94
[   16.562962] Bluetooth: hci0: BCM: features 0x2e
[   16.564412] Bluetooth: hci0: BCM43430A1
[   16.564452] Bluetooth: hci0: BCM43430A1 (001.002.009) build 0000
[   16.601408] Bluetooth: hci0: BCM43430A1 'brcm/BCM43430A1.hcd' Patch
[   17.304646] Bluetooth: hci0: BCM4343WA1 37.4MHz Murata Type-1DX BT4.2-0093
[   17.304689] Bluetooth: hci0: BCM43430A1 (001.002.009) build 0395
[   17.309939] cfg80211: Loading compiled-in X.509 certificates for regulatory database
[   17.881198] cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
[   18.289537] brcmfmac: brcmf_fw_alloc_request: using brcm/brcmfmac43430-sdio for chip BCM43430/1
[   18.501726] brcmfmac: brcmf_fw_alloc_request: using brcm/brcmfmac43430-sdio for chip BCM43430/1
[   18.571660] brcmfmac: brcmf_c_preinit_dcmds: Firmware: BCM43430/1 wl0: Mar 30 2021 01:12:21 version 7.45.98.118 (7d96287 CY) FWID 01-32059766
```

I made big changes in the project, see develop branch
Now the Linux kernel from the official ST repository with fresh updates and fixes: 5.10.61, latest u-boot and secure trust zone
And finally, the cherry on the cake - Ubuntu 22.04 
Te

![ScreenShot 2022-05-17 в 20 47 23](https://user-images.githubusercontent.com/26421560/168878023-62e2519d-89f4-4cfd-828a-b877e2240b05.png)
You can download `ubuntu-22.04-stm32mp1.img` from the latest build GitHub actions and help me with testing
